RetailSuiteAccounting/Integration/08_ManageCostAccounting.md

```diff
@@ -0,0 +1,247 @@
+[!!User Interface Cost accounting](../UserInterface/02e_CostAccounting.md)  
+
+# Manage the cost accounting
+
+Cost accounting is a method to identify, measure and record all costs associated with a company's production process. It serves as an internal controlling tool to assess the different business costs and determine profitability by department, producer, client, product, or other criteria.   
+
+A cost center is a department or a function within a company to which costs may be allocated, such as HR, Accounting or IT. It can also be defined at a smaller level according to management needs, for instance involving a particular job position or machine.  
+
+A cost object is a product or service in relation to which costs are ascertained. The cost object is determined by the nature of business. It can be the final item produced or a part of it, but can also be expressed differently, for instance in batches of items.  
+
+You can create, edit and delete both cost centers and cost objects.
+
```

Comment:
While posting an accounting entry you can either fill in cost center (Kost1) or cost object (Kost2).

 
![image](https://user-images.githubusercontent.com/118272226/212322700-d6fd2416-d4ae-482d-a5a2-afe849debee4.png)